client/app/screens/Login.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import Login from 'react-native-login';
+import Icon from 'react-native-vector-icons/FontAwesome';
```

Comment:
Actually, taking it a step further, these buttons should probably just be native-base Icon Buttons: https://docs.nativebase.io/COMPONENTS.html#Icon_Button.